pkg/azuredisk/controllerserver_v2.go

```diff
@@ -282,8 +299,18 @@ func (d *DriverV2) ControllerUnpublishVolume(ctx context.Context, req *csi.Contr
 		mc.ObserveOperationWithResult(isOperationSucceeded, consts.VolumeID, diskURI, consts.Node, string(nodeName))
 	}()
 
+	// attachMc := metrics.NewMetricContext(d.cloudProvisioner.GetMetricPrefix(), "detach_volume_latency", d.cloudProvisioner.GetCloud().ResourceGroup, d.cloudProvisioner.GetCloud().SubscriptionID, d.Name)
```

Comment:
If we aren't going to be using `detach_volume_latency`, let's get rid of the commented lines. If we plan to use it but are deferring for futuer PR, let's add `TODO` with a short description in the comment area.

Reply:
I think it would be best for now to remove the commented lines of code. As discussed, the existing `controller_unpublish_volume` metric covers the detach metric latency as well.